doc/installation/windows.md

```diff
@@ -39,7 +39,7 @@ Set two environment variables:
 set http_proxy=domain\username:password@proxy_server:port
 set https_proxy=domain\username:password@proxy_server:port
 ```
-If the Password contains special characters like `@,$,!` (e.g. `password:p@ssword`) then replace the special characters by their hex code equivalents with `%` prefix, as exemplified below:

```

Comment:
this won't make the test fail?  the password won't be complex enough

Reply:
the credscan log says that it failed in doc\installation\windows.md(42,)